src/components/common/Select/index.tsx

```diff
@@ -104,24 +143,15 @@ export default function Select(props: Props) {
                 {props.title}
               </Text>
             )}
-            <div className='w-full overflow-y-scroll max-h-70 scrollbar-hide'>
-              {props.options
-                .sort((a, b) =>
-                  (a.amount?.toNumber() ?? 0) > (b.amount?.toNumber() ?? 0) ? -1 : 1,
-                )
-                .map((option: SelectOption, index: number) => (
-                  <Option
-                    key={index}
-                    {...option}
-                    isSelected={
-                      option?.value
-                        ? option?.value === selected?.value
-                        : option?.denom === selected?.denom
-                    }
-                    onClick={handleChange}
-                  />
-                ))}
-            </div>
+            {isAssetsLoading ? (
+              <div className='flex justify-center items-center p-4'>
```

Comment:
Increase padding to make the overlay look a bit less narrow
```suggestion
              <div className='flex justify-center items-center px-4 py-10'>
```